helm/hpcc/values.schema.json

```diff
@@ -953,11 +1005,29 @@
           "type": "integer",
           "default": 1
         },
-        "servicePort": {
-          "type": "integer"
-        },
-        "port": {
-          "type": "integer"
+        "service": {
```

Comment:
I think this service section is identical to esp's apart from no "public" field - can it be commoned up?

Reply:
The lack of public is enough reason to not want to

Reply:
Does this change now you have local/global?

Reply:
I don't think so - we don't want to allow sasha services to be made visible.